ory_mocking/Consent/consent.go

```diff
@@ -0,0 +1,50 @@
+package Consent
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+type OAuth2ConsentResponse struct {
+	Challenge string `json:"challenge"`
+}
```

Comment:
Why do we need to define these structs, can't we use the ones in the hydra/kratos sdk?

`hydra_client.OAuth2ConsentRequest` seems to be the same struct. The same is true for the rest of the response structs.

Reply:
I replaced most of the inhouse structs. I kept the ones relating to error messages. The ory libraries store error messages as map[string]interface{} structs, and I already wrote the more comfortable structs. I would use the openapi error constructs in the main.go server.